docs/index.md

```diff
@@ -19,3 +19,7 @@ release X this protocol is fully supported.
 
 [communication]: https://wiki.diasporafoundation.org/How_we_communicate
 [entities]: {{ site.baseurl }}/entities/
```

Comment:
Can you add your block above line 20 and add the link to github to this list:
```[github]: https://github.com/diaspora/diaspora_federation```
and use it with `[on Github][github]` in your text.